module5/lessons/Week5/LinkedInActivityPD.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: LinkedIn Activity
+---
+
+Today is all about returning to your LinkedIn. You will be polishing up your profile and engaging with the broader developer community to make sure your LinkedIn best represent you!
+
+## Careerflow Competition! 
+
+Instructions for all team members:
+
+1. Install the [Careerflow Chrome Extension](https://www.google.com/url?q=https://chrome.google.com/webstore/detail/careerflow-ai-linkedin-op/iadokddofjgcgjpjlfhngclhpmaelnli&sa=D&source=calendar&ust=1695826733869948&usg=AOvVaw03e0L95JHMmx0VCrDyAm3g)
+2. Go to your Linkedin profile and click the blue box with a “C” in it on the right hand side of the page.
+    * It will prompt you to make an account to view suggested improvements.
+3. Make the recommended changes to your profile to improve it - you have until lunch on Thursday to make changes
+4. Add up the percentages of each team member and average them.
+
+The team spokesperson will DM your instructors with your group’s average score and any bonus points earned (Each bonus point counts as adding a percentage point to your team average!). Average score is due at noon this Thursday!
+
+The team with the highest average overall score will be the winners!
+
+### Bonus Points!

```

Comment:
I tried out the CareerFlow thing and it's pretty cool! This is what the output looks like.
<img width="185" alt="Screenshot 2023-10-19 100922" src="https://github.com/turingschool/launch-curriculum/assets/11747682/e1ec6dd9-55ee-4f00-a687-e2c967cfa32f">

Any other bonus things you can think of that we'd want them engaging with?

Reply:
<img width="173" alt="Screenshot 2023-10-19 101016" src="https://github.com/turingschool/launch-curriculum/assets/11747682/7bfe83f8-99e8-445e-944a-668db565070f">